DOCS/man/input.rst

```diff
@@ -2833,6 +2833,9 @@ Property list
 ``focused``
     Whether the window has focus. Might not be supported by all VOs.
 
+``ambient-light-lux``
```

Comment:
Do we need `-lux` suffix? Open question, I prefer simpler names, but I don't have strong opinion on this one.

Reply:
i am totally fine with leaving it out.